### DISCUSSION TITLE
Copilot seems to be causing major hang problems now

### DISCUSSION THREAD

**Discussion by TheSecurityDev** [community/community#8987] [Copilot Conversations] [answered]
After the last VSCode update (1.63.2), I've noticed that all extensions (I think) hang for about 30 seconds or so, quite frequently. The source control panel just stays loading (that's how I can tell when it's hanging because the clock icon stays there for several seconds), and Prettier hangs and my autocomplete doesn't work (or any other language features). I tried disabling Copilot and the issue went away. I re-enabled it and it came back immediately. I don't see any messages in the console about it though or anything.

I knew Copilot was amazing and all, but I didn't realize how much it was helping me until I had to turn it off :(

Specs:
```
Copilot v1.7.4273
VSCode Version: 1.63.2
OS: Windows_NT x64 10.0.19043
```

**Comment by Krzysztof-Cieslak** [comment, marked as answer]
Discussed/tracked in https://github.com/github/feedback/discussions/8861
